com.unity.ml-agents/CHANGELOG.md

```diff
@@ -8,24 +8,20 @@ and this project adheres to
 
 ## [Unreleased]
 ### Major Changes
+#### com.unity.ml-agents (C#)
```

Comment:
For the changes to master, maybe best to just change the `[Unreleased]` section. The 1.0.0 section might change more on the release branch (I still have a todo to fix the PR links), so it could get messy with merge conflicts when we merge the release branch back.

Reply:
alright, i'll just close these two and start over